Whiskey/Tasks/New-WhiskeyGitHubRelease.ps1

```diff
@@ -0,0 +1,186 @@
+        return
+    }
+
+    $baseUri = [uri]'https://api.github.com/repos/{0}' -f $repositoryName
+
+    [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor [System.Net.SecurityProtocolType]::Tls12
+
+    function Invoke-GitHubApi
+    {
+        [CmdletBinding(DefaultParameterSetName='NoBody')]
+        param(
+            [Parameter(Mandatory=$true)]
+            [uri]
+            $Uri,
+
+            [Parameter(Mandatory=$true,ParameterSetName='FileUpload')]
+            [string]
+            $ContentType,
+
+            [Parameter(Mandatory=$true,ParameterSetName='FileUpload')]
+            [string]
+            $InFile,
+
+            [Parameter(Mandatory=$true,ParameterSetName='JsonRequest')]
+            $Parameter,
+
+            [Microsoft.PowerShell.Commands.WebRequestMethod]
+            $Method = [Microsoft.PowerShell.Commands.WebRequestMethod]::Post
+        )
+
+        $optionalParams = @{ }
+        if( $PSCmdlet.ParameterSetName -eq 'JsonRequest' )
+        {
+            if( $Parameter )
+            {
+                $optionalParams['Body'] = $Parameter | ConvertTo-Json
+    	        Write-WhiskeyVerbose -Context $TaskContext -Message $optionalParams['Body']
+            }
+            $ContentType = 'application/json'
+        }
+        elseif( $PSCmdlet.ParameterSetName -eq 'FileUpload' )
+        {
+            $optionalParams['InFile'] = $InFile
+        }
+
+        try
+        {
+	        Invoke-RestMethod -Uri $Uri -Method $Method -Headers $headers -ContentType $ContentType @optionalParams
+        }
+        catch
+        {
+            if( $ErrorActionPreference -eq 'Ignore' )
+            {
+                $Global:Error.RemoveAt(0)
+            }
+            Stop-WhiskeyTask -TaskContext $TaskContext -Message ('GitHub API call to "{0}" failed: {1}' -f $uri,$_)
+        }
+    }
+
+    $tag = $TaskParameter['Tag']
+    if( -not $tag )
+    {
+        Stop-WhiskeyTask -TaskContext $TaskContext -Message ('Property "Tag" is mandatory. It should be the tag to create in your repository for this release. This is usually a version number. We recommend using the `$(WHISKEY_SEMVER2_NO_BUILD_METADATA)` variable to use the version number of the current build.')
+        return
+    }
+    $release = Invoke-GitHubApi -Uri ('{0}/releases/tags/{1}' -f $baseUri,[uri]::EscapeUriString($tag)) -Method Get -ErrorAction Ignore
+
+    $createOrEditMethod = [Microsoft.PowerShell.Commands.WebRequestMethod]::Post
+    $actionDescription = 'Creating'
+    $createOrEditUri = '{0}/releases' -f $baseUri
+    if( $release )
+    {
+        $createOrEditMethod = [Microsoft.PowerShell.Commands.WebRequestMethod]::Patch
+        $actionDescription = 'Updating'
+        $createOrEditUri = $release.url
+    }
+
+	$releaseData = @{
+			            tag_name = $tag

```

Comment:
What were you thinking on formatting?

Reply:
Oh I was just referring to the misalignment of brackets and extra spaces in this block:
```
	$releaseData = @{
			            tag_name = $tag
		            }
```
It might be just how github is displaying it though, or there's tabs mixed with spaces in there.

Reply:
Ah. I converted all the tabs in this file to spaces.